helper-scripts/test-local.fish

```diff
@@ -0,0 +1,38 @@
+#!/usr/local/bin/fish
+# A test script I use to test on my local devices
+
+echo 'Add this to objc2/Cargo.toml: `default = ["exception", "verify_message", "catch_all"]`'
+read
+
+export MACOSX_DEPLOYMENT_TARGET=10.7
+export IPHONEOS_DEPLOYMENT_TARGET=7.0
+
+# Use custom branch of cargo-dinghy that fixes an issue with iOS 9.3
+cargo install --git https://github.com/madsmtm/dinghy --branch fix-active-session --bin cargo-dinghy
+
+# Start the simulator
+open /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app
+
+# Test on the simulator
+cargo dinghy --device sim test
+cargo dinghy --device sim test --release
+
+# Test on my iPad mini 1st generation iOS 9.3
+# Followed this guide to set it up:
+# https://github.com/sonos/dinghy/blob/main/docs/ios.md
+#
+# We use build-std because the target is armv7-apple-ios
+mkdir .cargo
+echo '[unstable]' > .cargo/config.toml
```

Comment:
I'm mildly surprised that the [`cargo` manifest doesn't have target specific unstable features](https://doc.rust-lang.org/cargo/reference/unstable.html).

Reply:
Yeah, that would have been useful!

Just for reference, the reason I did it this way here is because `cargo dinghy` doesn't support `-Zbuild-std`.